composer.json

```diff
@@ -41,7 +41,8 @@
         "league/omnipay": "3.0.1",
         "academe/omnipay-authorizenetapi": "3.0.4",
         "omnipay/stripe": "3.0.1",
-        "waryway/php-traits-library": "^1.0"
+        "waryway/php-traits-library": "^1.0",
+        "paragonie/multi-factor": "0.2.2"
```

Comment:
I wonder if paragonie multi-factor is still being maintained. Last commit there was September 2017. What are other projects using?